src/aggregate.ts

```diff
@@ -1,26 +1,52 @@
 
-export enum AggregateOp {
-    VALUES = 'values' as any,
-    COUNT = 'count' as any,
-    VALID = 'valid' as any,
-    MISSING = 'missing' as any,
-    DISTINCT = 'distinct' as any,
-    SUM = 'sum' as any,
-    MEAN = 'mean' as any,
-    AVERAGE = 'average' as any,
-    VARIANCE = 'variance' as any,
-    VARIANCEP = 'variancep' as any,
-    STDEV = 'stdev' as any,
-    STDEVP = 'stdevp' as any,
-    MEDIAN = 'median' as any,
-    Q1 = 'q1' as any,
-    Q3 = 'q3' as any,
-    MODESKEW = 'modeskew' as any,
-    MIN = 'min' as any,
-    MAX = 'max' as any,
-    ARGMIN = 'argmin' as any,
-    ARGMAX = 'argmax' as any,
+export namespace AggregateOp {
+    export const VALUES: 'values' = 'values';
```

Comment:
@domoritz Yes, but the downside is that `AggregateOp.ARGMAX` (and so on) is no longer usable bare in a type position as it was when it was an enum. Is that a fair tradeoff?